src/content/docs/it/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,423 @@
+
+Nella v1.x, l'opzione `extendPlugins` dell'integrazione MDX gestiva come i tuoi file MDX dovevano ereditare le configurazioni di Markdown: tutte le tue configurazioni di Markdown (`markdown`), o solamente i plugin predefiniti di Astro (`default`).
+
+Astro v2.0 sostituisce questo comportamento controllato da `mdx.extendPlugins` con tre nuove opzioni, configurabili indipendentemente, che sono impostate a `true`:
+
+- **[`mdx.extendMarkdownConfig`](/it/guides/integrations-guide/mdx/#extendmarkdownconfig)** to inherit all or none of your Markdown configuration
+- **`mdx.gfm`** to enable or disable GitHub-Flavored Markdown in MDX
+- **`mdx.smartypants`** to enable or disable SmartyPants in MDX
+
+##### Cosa devo fare?
+
+Elimina `extendPlugins: 'markdown'` nella tua configurazione. Ora questo comportamento è predefinito.
+
+```js del={8}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  integrations: [
+    mdx({
+      extendPlugins: 'markdown',
+    }),
+  ],
+});
+```
+
+Sostituisci `extendPlugins: 'defaults'` con `extendMarkdownConfig: false` e aggiungi le opzioni separate per GitHub-Flavored Markdown e SmartyPants per abilitare indipendentemente questi plugin predefiniti in MDX.
+
+```js del={8} ins={9-11}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  integrations: [
+    mdx({
+      extendPlugins: 'defaults',
+      extendMarkdownConfig: false,
+      smartypants: true,
+      gfm: true,
+    }),
+  ],
+});
+```
+
+#### Aggiunto: Maggiori opzioni di configurazione di MDX per stare al passo con Markdown
+
+Astro v2.0 ti permette ora di impostare individualmente [ogni possibile opzione di configurazione di Markdown](/it/reference/configuration-reference/#markdown-options) (eccetto `drafts`) separatamente nella configurazione della tua integrazione MDX.
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  markdown: {
+    remarkPlugins: [remarkPlugin1],
+    gfm: true,
+  },
+  integrations: [
+    mdx({
+      remarkPlugins: [remarkPlugin2],
+      gfm: false,
+    })
+  ]
+});
+```
+
+##### Cosa devo fare?
+
+Rivedi le configurazioni di Markdown e MDX e confronta la tua configurazione con le nuove opzioni disponibili.
+
+
+### Cambiato: Accesso dei plugn all'avantesto
+
+Nella v1.x, i plugin remark e rehype non avevano accesso all'avantesto dell'utente. Astro ha unito l'avantesto del plugin con l'avantesto del tuo file, senza dover passare l'avantesto del file ai tuoi plugin.
+
+Astro v2.0 consente ai plugin remark e rehype l'accesso all'avantesto degli utenti tramite l'iniezione di avantesto. Ciò consente agli autori di plug-in di modificare l'avantesto esistente di un utente o di calcolare nuove proprietà basate su altre proprietà.
```

Comment:
**nitpick**: I would use `inserimento` instead of `iniezione`, it seems more fluent.